services/query.proto

```diff
@@ -200,8 +193,17 @@ message Query {
         GetAccountDetailsQuery accountDetails = 58;
 
         /**
-         * Get all information about a node
+         * Request detail information for a consensus node within the network
+         * address book.
+         * <p>
+         * This query is a privileged action that requires signature from the
+         * Hedera governing council.<br/>
+         * The response, when successful, SHALL contain the full information
+         * currently available, including current status, for the single
+         * consensus node identifier provided.<br/>
+         * The node queried MAY be active, inactive, deleted, or pending a
+         * change at the next network upgrade.
          */
-        NodeGetInfoQuery nodeGetInfo = 59;
+        com.hedera.hapi.node.addressbook.NodeGetInfoQuery nodeGetInfo = 59;
```

Comment:
Don't need the whole package name here.

Reply:
It is required now; as the package directive in the files is updated.